Gemfile.lock

```diff
@@ -96,6 +96,13 @@ GEM
     cocaine (0.6.0)
       terrapin (= 0.6.0)
     coderay (1.1.2)
+    coffee-rails (4.2.2)
```

Comment:
Ideally, we shouldn't be using coffescript in our dependencies. Looks like this is coming from rails_admin :/ We using a lot of dependencies just to be able to view insights into reports. 

I really recommend writing our own admin dashboard, that way we don't have to bring in more unnecessary dependencies. All we have to do is set up a feature flag for admin users in Devise and then build pages where only those users can view data.

Reply:
Devise has official documentation for this: https://github.com/plataformatec/devise/wiki/How-To:-Add-an-Admin-Role